<!--
Setup kinematics simulation of Motoman SIA10f.
-->

<launch>

  <include file="$(find ay_util)/launch/motoman/motomansia10f_cmn.launch"/>

  <node pkg="ay_util" type="dummy_robot.py"
        name="moto_dummy" output="screen"
        args="100" >
    <remap from="/follow_joint_trajectory" to="/joint_trajectory_action" />
  </node>

  <node pkg="rviz" type="rviz" name="rviz"/>

</launch>
